README.md

```diff
@@ -8,21 +8,32 @@ The various configuration frameworks are intialised by supplying appsetting.json
 # Example Usage
 
 ```
-[Binding]
+[   [Binding]
     public class ContactUsConfigurationSetup
     {
         private readonly ScenarioContext _context;
         private readonly ObjectContext _objectContext;
         //private readonly IConfigurationRoot _configurationRoot;
+        //private readonly IConfigSection _configSection;
```

Comment:
Did you leave this comment in intentionally?

Reply:
removed